public/styles/components/_chart.scss

```diff
@@ -23,24 +19,20 @@ $loadingLabelHeight: 300px;
   right: 0;
 }
 
-.chart-wrap__updating:after,
-.chart-wrap__updating:before {
-  display: block;
+.chart-wrap:after,
+.chart-wrap__error:after {
```

Comment:
If you also add `chart-wrap` in the error state, you won't need to add the modifier class here :)